packages/flutter/lib/src/services/binding.dart

```diff
@@ -101,7 +101,13 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
     // TODO(ianh): Remove this complexity once these bugs are fixed.
     final Completer<String> rawLicenses = Completer<String>();
     scheduleTask(() async {
-      rawLicenses.complete(rootBundle.loadString('LICENSE', cache: false));
+      // TODO(jonahwilliams): temporary catch to allow migrating LICENSE to NOTICES.
```

Comment:
nit: can you link this to an issue so we know it's safe to remove when the issue is closed?

Reply:
The issue spans a few different changes required, like updating the engine/framework licenses. WDUT about linking to the PR? (https://github.com/flutter/flutter/pull/57871)

Reply:
PR's perfect!

Reply:
Done